Services/Membership/classes/class.ilParticipant.php

```diff
@@ -287,6 +289,13 @@ public function isParticipant()
     
     public function getNumberOfMembers()
     {
+        global $DIC;
+
+        $rbacreview = $DIC['rbacreview'];
+
+        if (!$this->numMembers) {
```

Comment:
@Amstutz  / @smeyer-ilias Maybe `if (null === $this->numMembers)` should be used here. The result of `$rbacreview->getNumberOfAssignedUsers($this->member_roles);` can `equal 0`, and `!$this->numMembers` would evaluate in a boolean `true`.

Reply:
Yep, will be fixed by an extra commit.